week-4-practise/level-2/CODEWARS.md

```diff
@@ -0,0 +1 @@
+- https://www.codewars.com/kata/welcome/train/javascript

```

Comment:
Perhaps there should be an introduction for students who just open the file, like "Complete the following Codewars exercises. Go to the webpage below and follow the instructions there."

Reply:
that makes sense! I'll add something in.